src/games/strategy/triplea/ui/screen/IDrawable.java

```diff
@@ -128,26 +128,29 @@ public void draw(final Rectangle bounds, final GameData data, final Graphics2D g
         return;
       }
     }
-    final Rectangle territoryBounds = mapData.getBoundingRect(territory);
+
+    // get font metrics
     graphics.setFont(MapImage.getPropertyMapFont());
     graphics.setColor(MapImage.getPropertyTerritoryNameAndPUAndCommentcolor());
     final FontMetrics fm = graphics.getFontMetrics();
+
+    // if we specify a placement point, use it otherwise try to center it
     int x;
     int y;
-    // if we specify a placement point, use it
-    // otherwise, put it in the center
     final Point namePlace = mapData.getNamePlacementPoint(territory);
     if (namePlace == null) {
+      final Rectangle territoryBounds = mapData.getBestTerritoryNameRect(territory, fm);
```

Comment:
`mapData.getBoundingRect(territory);` is called in other locations, it's a bit odd this is the only spot that benefits. If that is really the case, why not make this a private method attached to this class that uses it, and have it accept a mapData object?